tests/web/test_websocket_server.py

```diff
@@ -154,15 +156,16 @@ def test_websocket_string_event(test_client, loop):
 
 
 @asyncio.coroutine
-def test_websocket_block_event(b, test_client, loop):
+@pytest.mark.parametrize('_block', (10,), indirect=('_block',), ids=('block',))
+def test_websocket_block_event(b, _block, test_client, loop):
     from bigchaindb import events
     from bigchaindb.web.websocket_server import init_app, POISON_PILL, EVENTS_ENDPOINT
 
     event_source = asyncio.Queue(loop=loop)
     app = init_app(event_source, loop=loop)
     client = yield from test_client(app)
     ws = yield from client.ws_connect(EVENTS_ENDPOINT)
-    block = create_block(b, 10).to_dict()
+    block = _block.to_dict()
```

Comment:
General note for this commit: 

Using a fixture is proposed here instead of a function because we already have some tests that also rely on having a block, and these tests also have their own function to create the dummy block. See https://github.com/bigchaindb/bigchaindb/blob/master/tests/db/test_bigchain_api.py#L26-L31 for instance.

We now have the opportunity to take care of this, hence this proposed change. The fixture could perhaps renamed differently, and moved up into the main `conftest` module.